ern-core/src/android.ts

```diff
@@ -248,8 +263,9 @@ export async function runAndroidUsingAvdImage({
 }) {
   // https://issuetracker.google.com/issues/37137213
   spawnp(androidEmulatorPath(), ['-avd', avdImageName], { detached: true })
-
-  await kax.task('Waiting for device to start').run(waitForAndroidDevice())
+  await kax
+    .task('Waiting for device to start')
+    .run(Promise.race([waitForAndroidDevice(), failAfterTimeOut(60000)]))
```

Comment:
Not 100% sure how it works, but what if for instance on a very slow computer, the emulator takes >60s to start up?

Reply:
The goal is to not wait forever, Couldn't find the default value set for the timeout in the Android source. except one can override value https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.internal.dsl.AdbOptions.html

Is there any other value you'd recommend?

Reply:
I don't really have a value I would recommend.. So I guess I'm OK with 60s ;)
Can you also please check the comments I left on #1489, since they're somewhat related. If you agree, see if you can repurpose the PR. Or otherwise, we can merge it and we'll address it as a follow-up.

On a more general level, the question is, do we really need code in ERN that waits for an emulator to start? In my opinion, we should ideally **remove this kind of complexity** that is not a concern of ERN. React Native also does not care about it, and I have not seen anyone complain about it. Is there a strong reason we need to?

Reply:
I read comments #1489 I believe the PR is in line with the problem reported.
At the moment we surely have the logic to wait for the AVD indefinite time if `init.svc.bootanim` doesn't reach the `stopped` state.

Secondly, if there is no emulator configured we should fail with the proper message, the logic was missing and the PR fixes it. 

I tried, 
- No configured AVD (fails with the proper message)
- The emulator doesn't reach the state `stopped` (fails with the proper error message)
- The happy path where the mini-app runner

Reply:
It does now, yes. `a0d0cd5` didn't have the change in `getAndroidAvds` yet. ;) :+1: